src/utils.js

```diff
@@ -6,7 +6,32 @@ function remove(array, element) {
   array.splice(array.indexOf(element), 1);
 }
 
+const mockGithub = {
```

Comment:
But this mock in the spec file instead of the utils file as it is just used in tests.